proposals/0099-8-bit-types.md

```diff
@@ -0,0 +1,67 @@
+<!-- {% raw %} -->
+
+# Refined `cbuffer` Contexts
+
+## Instructions
+
+* Proposal: [0099](0099-8-bit-types.md)
+* Author(s): [Matthäus Chajdas](https://github.com/anteru)
+* Sponsor: TBD
+* Status: TBD
```

Comment:
Could you set the sponsor to `[Chris Bieneman](https://github.com/llvm-beanz)` please.  The status should be "Under Consideration".

Chris can help get a proposal number assigned and everything else required to get the PR merged.

Reply:
Done, thanks!